src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/payment/PaymentsCallbackPBAHandler.java

```diff
@@ -0,0 +1,149 @@
+import static uk.gov.hmcts.reform.civil.enums.PaymentStatus.SUCCESS;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class PaymentsCallbackPBAHandler extends CallbackHandler {
+
+    private static final List<CaseEvent> EVENTS = Collections.singletonList(MAKE_PBA_PAYMENT);
+    private static final String ERROR_MESSAGE = "Technical error occurred";
+    private static final String TASK_ID = "CreateClaimMakePayment";
+    public static final String DUPLICATE_PAYMENT_MESSAGE
+        = "You attempted to retry the payment to soon. Try again later.";
+
+    private final PaymentsService paymentsService;
+    private final ObjectMapper objectMapper;
+    private final FeatureToggleService featureToggleService;
+    private final Time time;
+
+    @Override
+    public String camundaActivityId(CallbackParams callbackParams) {
+        return TASK_ID;
+    }
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return Map.of(
+            callbackKey(ABOUT_TO_SUBMIT), this::makePbaPayment
+        );
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        if (featureToggleService.isPbaV3Enabled()) {
+            return EVENTS;
+        } else {
+            return Collections.emptyList();
+        }
+    }
+
+    private CaseData updateWithDuplicatePaymentError(CaseData caseData) {
+        var paymentDetails = ofNullable(caseData.getClaimIssuedPaymentDetails())
+            .map(PaymentDetails::toBuilder)
+            .orElse(PaymentDetails.builder())
+            .status(FAILED)
+            .errorCode(null)
+            .errorMessage(DUPLICATE_PAYMENT_MESSAGE)
+            .build();
+
+        return caseData.toBuilder().claimIssuedPaymentDetails(paymentDetails).build();
+    }
+
+    private CallbackResponse makePbaPayment(CallbackParams callbackParams) {
+        var caseData = callbackParams.getCaseData();
+        var authToken = callbackParams.getParams().get(BEARER_TOKEN).toString();
+        List<String> errors = new ArrayList<>();
+        try {
+            log.info("processing payment for case " + caseData.getCcdCaseReference());
+            var paymentReference = paymentsService.createCreditAccountPayment1(caseData, authToken)
+                .getPaymentReference();
+
+            PaymentDetails paymentDetails = ofNullable(caseData.getClaimIssuedPaymentDetails())
+                .map(PaymentDetails::toBuilder)
+                .orElse(PaymentDetails.builder())
+                .status(SUCCESS)
+                .reference(paymentReference)
+                .errorCode(null)
+                .errorMessage(null)
+                .build();
+
+            caseData = caseData.toBuilder()
+                .claimIssuedPaymentDetails(paymentDetails)
+                .paymentSuccessfulDate(time.now())
+                .build();
+
+        } catch (FeignException e) {
+            log.info(String.format("Http Status %s ", e.status()), e);
```

Comment:
This is an application error. This should be at `error` level.

Also, no need to use `String.format`. It can simply be `log.error("Http Status {} ", e.status(), e);`

BONUS: web applications are multi-threaded by nature so logs can be interspersed. Along with the http status, also log `caseData.getCcdCaseReference()`.

Reply:
Changed to suggested format - resolved myself, as you are still off on sick leave.